cluster/ci/jobs/image-mirror.yaml

```diff
@@ -234,3 +234,144 @@ items:
               - mirror
               - -f=/etc/imagemirror/origin_4_2
               - --skip-multiple-scopes # quay chokes when requesting multiple
+- apiVersion: batch/v1beta1
```

Comment:
Why is this PR adding some knative jobs? Is it intentional?

Reply:
Good point, I don't think it should add anything in there, especially related to knative…

Reply:
100% agree

I did actually factor the tekton bits... out ->   https://github.com/arilivigni/release/pull/1/files

Reply:
I will remove those apologies